packages/react-core/src/components/SearchInput/SearchInput.tsx

```diff
@@ -277,11 +285,9 @@ const SearchInputBase: React.FunctionComponent<SearchInputProps> = ({
 
   if (!!onSearch || attributes.length > 0 || !!onToggleAdvancedSearch) {
     if (attributes.length > 0) {
-      return (
-        <div className={css(className, styles.searchInput)} ref={searchInputRef} {...props}>
-          {buildSearchTextInputGroupWithExtraButtons()}
+      const AdvancedSearch = (
+        <span>
```

Comment:
@bartoval Apologies for the super late comment, I was following up on something I made a note about a while ago. FWIW with the original issue, we should always try and avoid applying multiple component styles to an element (ie, `.pf-c-panel.pf-c-search-intput__menu`). 

This element should be a `<div>` though, a `<span>` is an inline element and its permitted content is [phrasing content](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories#phrasing_content). So technically it's invalid HTML to put non-phrasing content in a `<span>`, though it works just fine. Created an issue for that here - https://github.com/patternfly/patternfly-react/issues/7796

Also it's worth noting that `appendTo="inline"` just drops the panel in the layout and pushes everything below it down (it doesn't display on top of the stuff below it). A few questions around that:
* Should `appendTo` be `menuAppendTo` like the other components that have a menu in them that popper can position?
* Is that the intended behavior of `appendTo="inline"` or should that display on top of the stuff below it like it used to - it just won't use popper. Not sure how we'd do that if we want to continue to use the panel component there, but that would be inline with the other menus that support `menuAppendTo`. 
* If we are going to leave it so that it pushes the stuff below the panel down, should we remove the "raised" modifier from the panel since the shadow implies depth/overlay?

Reply:
@mcoker thanks for your comments! They are very interesting.

- I will change the markup span in div

Regarding your questions:
- there is comment above about that and using menuAppendTo is a little misleading since the Menu is a component we have and we are not using a Menu in this case.

- Yes this behaviour is predicted like the TimePicker (@nicolethoen )

-  In my opinion the shadow is not associated to an overlay. Ie: A card have a shadow but it doesn't implies overlay.  However this is just a personal tought.

Reply:
@bartoval thanks!! 

> there is comment above about that and using menuAppendTo is a little misleading since the Menu is a component we have and we are not using a Menu in this case.

👍 got it. I agree, that could be confusing. I would probably argue that menu is more of a concept there to refer to the thing that pops open in the component, and that could be used with whatever component pops up. It seems like it might be more predictable/consistent that way. As in if we stopped using the "menu" component and used some other component for the thing that pops up, `menuAppendTo` would still be fine and wouldn't necessarily need to change IMO since we shouldn't be tying the prop name to the component used in the first place and we're using menu as a concept. Also I'd probably say that `[menu/thing]AppendTo` would be used on the component that has a thing that pops up, and `appendTo` would be more appropriate on the thing that pops up itself. Just my $0.02 😁